webapp/src/Form/Type/ContestType.php

```diff
@@ -39,28 +41,34 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'Yes' => true,
                 'No' => false,
             ],
+            'help' => 'Only John knows',
         ]);
         $builder->add('freezetimeString', TextType::class, [
             'label' => 'Scoreboard freeze time',
             'required' => false,
+            'help' => 'Time when the freeze starts: the results of submissions made after this time are not revealed until the scoreboard unfreeze time below has passed.',
         ]);
         $builder->add('endtimeString', TextType::class, [
             'label' => 'End time',
+            'help' => 'Time when the contest ends.',
         ]);
         $builder->add('unfreezetimeString', TextType::class, [
             'label' => 'Scoreboard unfreeze time',
             'required' => false,
+            'help' => 'Time when the final scoreboard is revealed. Usually a few hours after the contest ends, and the award ceremony is over.',
         ]);
         $builder->add('deactivatetimeString', TextType::class, [
             'label' => 'Deactivate time',
             'required' => false,
+            'help' => 'Time when the contest and scoreboard are hidden again. Usually a few hours/days after the contest ends.',
         ]);
         $builder->add('processBalloons', ChoiceType::class, [
             'expanded' => true,
             'choices' => [
                 'Yes' => true,
                 'No' => false,
             ],
+            'help' => 'Enables the balloons page. (What is the advantage of disabling it?)',
```

Comment:
>  (What is the advantage of disabling it?)

This was introduced by @nickygerritsen in 1f3d6d9181ec33127186e52cbba9e12b43ac16e2 but I don't remember why exactly. For this PR, let's just remove this part of the comment.

Reply:
If you use an external tool to process balloons this gives a minor speed improvement but i I don’t mind to drop it since it’s not that useful

Reply:
Not sure what you mean with external tool but if you disable this, the balloons table will not be filled so you can also not use an external tool I guess. I think it's indeed some kind of optimization that we do not fill another table if you don't do anything with it?

Reply:
I mean the cds. It can also generate balloons